articles/data-factory/control-flow-for-each-activity.md

```diff
@@ -478,6 +478,7 @@ Here are some limitations of the ForEach activity and suggested workarounds.
 |---|---|
 | You can't nest a ForEach loop inside another ForEach loop (or an Until loop). | Design a two-level pipeline where the outer pipeline with the outer ForEach loop iterates over an inner pipeline with the nested loop. |
 | The ForEach activity has a maximum `batchCount` of 50 for parallel processing, and a maximum of 100,000 items. | Design a two-level pipeline where the outer pipeline with the ForEach activity iterates over an inner pipeline. |
+| SetVariable can't be used inside a ForEach activity that runs in parallel as the variables are global to the whole pipeline, they are not scoped to a ForEach or any other activity. | Redesign the Pipeline to either use sequential ForEach (which might denigrate performance) or use Execute Pipeline inside ForEach (Variable/Parameter handled in child Pipeline).|
```

Comment:
Please change the text "Redesign the Pipeline ... " to "Consider using sequential ForEach or Execute Pipeline".

Reply:
Updated, please take a look.